apps/ideas/templates/advocate_europe_ideas/includes/call_to_action_tile.html

```diff
@@ -0,0 +1,18 @@
+{% load static i18n rules rule_tags %}
+<div class="call-to-action-tile idealist-item">
+    <div class="call-to-action-img">
+        <img src="{% static 'images/ae_logo_white.svg' %}">
+    </div>
+    <div>
+        <h4>{% trans "Got an idea? Share it!" %}</h4>
+        <div>{% trans "Get support from the community and win funding for your project!" %}</div>
+        <a class="call-to-action-link" href="#">{% trans "Read about the criterias" %}</a>
+    </div>
+    {% get_modules as modules %}
+    {% for module in modules %}
+        {% has_perm 'advocate_europe_ideas.add_ideasketch' request.user module as can_add %}
```

Comment:
We should also show the button to not-logged-in users! How can I only check if the phase is active? And what do we want to happen otherwise? Not show the whole tile, of course, so this logic has to wander to profile.html!